Adding hands-on

ChristinaLK opened the pull request:
Here's a first draft of the hands-on section of the proposal.  Questions for everyone: 
- Do we need more example exercises?  Or more specifics about the exercises?  
- What do people think of the two examples I wrote up?  If you have alternatives/suggestions, please share!
- @dbrunson, is the information I included about OK State + XSEDE resources accurate?  
- Anything else missing?

ChristinaLK committed:
initial draft of hands-on description

ChristinaLK committed:
completed first draft of hands-on for proposal

dbrunson commented:
Looks great!   You could put "education allocation" to be more specific, but I doubt that level of detail is terribly important.

shwina commented on hands-on.md:
```diff
@@ -1,2 +1,68 @@
+among participants to answer questions or address immediate issues.  
+ 
+Exercises and demonstrations will be fully tested on the planned resources 
+before the workshop. 
```
Etherpad?

shwina commented:
There is room for "visuals" of demos, that we can provide as an attachment, or as a link to a web page, etc. Do we want to add visuals of what the code boxes ("input", "output" and "error") look like in our lesson material? Or do you think this is too fine a detail?

shwina commented on hands-on.md:
```diff
@@ -1,2 +1,68 @@
+
+> **Example: Submitting jobs to a batch scheduler**
+> 
+> Participants modify a job submission file 
```
Maybe a concrete challenge/example would serve better here?

ChristinaLK replied:
What do you mean by more concrete?

shwina replied:
Sorry! I meant something like this maybe:

`submit-job.pbs` submits a request to run our script on the test file `test.csv`:

```
python analyze.py test.csv
```

Modify the script `submit-job.pbs` to instead submit a job to run the analysis on our actual input data, `temperatures.csv`. This job will need more than 1 GB, so you will also need to change the amount of memory that `submit-job.pbs` is requesting.

ChristinaLK replied:
That's great - I'll make that change.

ChristinaLK commented:
The most appropriate "visual" to include would be a link to the actual lesson material, which hasn't been developed yet and/or some diagrams of topics we'll be teaching on (which also don't exist :P).  

If appropriate, we could link to another Software Carpentry lesson page and say that "the lesson will look like this" (with exercises, and so on).

shwina commented:
I think it's appropriate to do that, just so reviewers have some idea about what exactly the "lesson material" refers to.

ChristinaLK committed:
incorporating Ashwin's suggestions:
- adding more concrete exercise
- adding link to SWC lesson materials as example

shwina commented:
Fantastic! Thanks @ChristinaLK
